src/ApiService/ApiService/onefuzzlib/Config.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ApiService.OneFuzzLib.Orm;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.OneFuzz.Service;
+
+
+public interface IConfigOperations : IOrm<InstanceConfig>
+{
+    Task<InstanceConfig> Fetch();
+
+    Task Save(ILogTracer log, InstanceConfig config, bool isNew, bool requireEtag);
+}
+
+public class ConfigOperations : Orm<InstanceConfig>, IConfigOperations
+{
+    private readonly IEvents _events;
+    public ConfigOperations(IStorage storage, IEvents events) : base(storage)
+    {
+        _events = events;
+    }
+
+    public async Task<InstanceConfig> Fetch()
+    {
+        var key = EnvironmentVariables.OneFuzz.InstanceName ?? throw new Exception("Environment variable ONEFUZZ_INSTANCE_NAME is not set");
+        var config = await GetEntityAsync(key, key);
+        return config;
+    }
+
+    public async Task Save(ILogTracer log, InstanceConfig config, bool isNew = false, bool requireEtag = false)
```

Comment:
So in the old instance_config app function init.py, the post is where we update the NSG. Are we thinking this would eventually go here? Or do we just need to refactor the instance_config app function entirely and that will be a separate file?

Reply:
so that Http Trigger for instance config is still to be written. 

That code will be in the HttpTrigger handler